scripts/cola_baseline.sh

```diff
@@ -12,3 +12,9 @@ python main.py --config_file "config/spring19_seminar/cola_elmo.conf" --override
 
```

Comment:
Mind moving this to a scripts/spring19-seminar directory?

Reply:
Will do.